test/kitchen/site-cookbooks/dd-agent-upgrade/recipes/default.rb

```diff
@@ -73,7 +73,7 @@
       owner 'root'
       group 'root'
       mode '0644'
-      content "deb [signed-by=#{apt_usr_share_keyring}] #{node['dd-agent-upgrade']['aptrepo']} #{node['dd-agent-upgrade']['aptrepo_dist']} #{node['dd-agent-upgrade']['agent_major_version']}"
```

Comment:
Is it ok to remove `[signed-by=#{apt_usr_share_keyring}]` ?

Reply:
It's removed here, but added in the value of `aptrepo` defined in `test/kitchen/test-definitions/platforms-common.yml`, to make this a noop.

Reply:
oh! it makes completely sense! Thanks for clarifying it!